addons/crm/wizard/crm_lead_pls_description.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools.translate import _
+
+
+class LeadPlsDescriptionField(models.TransientModel):
+    _name = 'crm.lead.pls.description.field'
+    _description = 'Information of the field used in the PLS calculation'
+
+    lead_pls_description_id = fields.Many2one('crm.lead.pls.description', readonly=True)
+
+    field_name = fields.Char('Name of the field', readonly=True)
+    field_value = fields.Char('String value of the field', readonly=True)
+
+    field_number_of_won = fields.Char('#Won', help='Number of won leads which have this value', readonly=True)
+    field_number_of_lost = fields.Char('#Lost', help='Number of lost leads which have this value', readonly=True)
+    field_total_won = fields.Char('Won leads', help='Total number of won leads', readonly=True)
+    field_total_lost = fields.Char('Lost leads', help='Total number of lost leads', readonly=True)
```

Comment:
a priori, this number will always be the same for each field. (except stage_id.. I don't think it's really important to show..
See with @LNAodoo )
It's mainly used during the computation to normalise the score.

Reply:
@dbeguin @LNAodoo this field is important to show I think, because it determine the percentage (bigger is ``total_won``, smaller will be the probability if ``number_of_won`` stay unchanged)